src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class should have an access modifier. In Java, it is best practice to define the access level explicitly. Consider adding the 'public' access modifier to the Ball class definition.